docs/fiddles/windows/manage-windows/New Window Fiddle/renderer.js

```diff
@@ -0,0 +1,13 @@
+const { BrowserWindow } = require('electron').remote
+const path = require('path')
+
+const newWindowBtn = document.getElementById('new-window')
+
+newWindowBtn.addEventListener('click', (event) => {
+  const modalPath = path.join('file://', __dirname, '../../sections/windows/modal.html')
```

Comment:
This wouldn't work since the modal file is not there, I have a local branch which has all four of the manage window sections ported, but since the modal files can't be embedded to Fiddle (it throws exception), I've hold on to that for now. Also asked a question from @erickzhao on the issue itself which hasn't been answered yet

Reply:
I'm not sure I follow, what exactly it wouldn't work? Because I tested it and it did open a new window, you mean wouldn't display the contents inside of the modal? Like the hello world dummy text in the green background?

Reply:
The Fiddle only has three files (`main.js`, `renderer.js`, and `index.html`), that modal file is not available and so when I was testing it was throwing exception. Unless you've run your Fiddle in a directory where `.../../sections/windows/modal.html` existed relative to it